cluster-http/src/main/scala/akka/management/cluster/ClusterHttpManagementRoutes.scala

```diff
@@ -143,4 +143,22 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementJsonProtocol {
       }
     )
 
+  /**
+   * Creates an instance of [[ClusterHttpManagementRoutes]] with only the read only routes.
+   * the specified path `pathPrefixName`.
```

Comment:
where is that `pathPrefixName` specified?

Reply:
Why remove private scope on all? I mean versus at least something a little less vs completely open. Are we now Exposing to users?

Reply:
That is done in akka-mangement when it hosts them so not valid. Will remove comemnt.

Reply:
I removed the privates in case users wanted a subset of them. I can add them back.